src/modm/platform/gpio/stm32/module.lb

```diff
@@ -237,6 +240,10 @@ def validate(env):
         all_peripherals.extend([get_driver(r) for r in driver["remap"]])
     bprops["all_peripherals"] = sorted(list(set(all_peripherals)))
     bprops["all_signals"] = sorted(list(set(s["name"] for s in all_signals.values())))
+
+    pp(bprops["all_peripherals"])
+    pp(bprops["all_signals"])
+
```

Comment:
modm-devices needs a much better interface for accessing the raw data and derived data. The data format is optimized for unambiguity, not accessiblity.